package.json

```diff
@@ -20,7 +20,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint . --ext .ts --ext .tsx"
```

Comment:
It would be good to merge the commitlint PR, so you can have the husky acting as commit hook and linting the staged files before commiting

Reply:
Furthermore: Shall we use the airbnb styleguide?

Reply:
I agree I will wait for the merging of commitlint and add a hook for linting

Reply:
Airbnb styleguide I don't know 🤷‍♂️ I can add it for sure 🙂